tests/ItemTest.php

```diff
@@ -67,4 +68,29 @@ public function testIsHit2()
 
         $this->assertFalse($item->isHit());
     }
+
+    public function testExpiresAt()
+    {
+        $item = new Item("test");
+
+        $this->assertInstanceOf(Item::class, $item->expiresAt(new \DateTime()));
+    }
+
+
+    public function expiresAfterDataProvider()
+    {
+        return [
+            [\DateInterval::createFromDateString('1 day')],
+            [10],
+        ];
+    }
+    /**
+     * @dataProvider expiresAfterDataProvider
+     */
+    public function testExpiresAfter($time)
+    {
+        $item = new Item("test");
+
+        $this->assertInstanceOf(Item::class, $item->expiresAfter($time));
```

Comment:
What's the purpose of this test? Shouldn't we be checking an appropriate expiry has been set?

Reply:
If we want to check expired time, I think we should implement the ```getExpiredTime``` method to check whether this is set expired time correctly.

Reply:
I use the ```assertGreaterThanOrEqual``` to check the expired time.